### PR TITLE
Homework Two

### PR DESCRIPTION
**Pull request by khomtali**

### CODE REVIEW

File: homework-1/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        <img src="img/gletscher-stubaital.jpg" alt="Stubai Glacier">
+        <figcaption>Stubai Glacier</figcaption>
+      </figure>
+      <p><a href="https://g.page/stubai-glacier?share" target="_blank">Location</a></p>
+      <p class="place-box__description">That's not a Munich place, but I miss snowy mountains of Tyrol sooo much. I was planning to snowboard on Easter, but... Unfortunately, I can't do it again till the next season.</p>
+    </section>
+    <section class="place-box">
+      <h2>Flaucherstandl Kiosk</h2>
+      <figure>
+        <img src="img/flaucher.jpg" alt="Flaucherstandl Kiosk">
```
Comment (MrTim) [inline homework-1/index.html:33]: I think you can put figurecaption here as well.

File: homework-1/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  <link href="css/style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Baloo+Thambi+2:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul class="menu">
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About Me</a></li>
+        <li class="active"><a href="#">Base Page</a></li>
```
Comment (MrTim) [inline homework-1/index.html]: I think li:active css selector should work in that case as well.

**Review by MrTim — COMMENTED** [review]
Nice work, added just couple comments.

File: homework-1/index.html (hunk 1 above)
Comment (khomtali) [reply homework-1/index.html:33]: thanks! line added below

File: homework-1/index.html (hunk 2 above)
Comment (khomtali) [reply homework-1/index.html]: thanks! class of "base page" link updated to be different, as a current page

**Review by MrTim — APPROVED** [review]
Fine for me! I think you can merge it.

**Pull request merged**